resources.rst

```diff
@@ -0,0 +1,37 @@
+=========
+Resources
+=========
+
+This document lists some of the resources used to understand the theory
+behind this work and other useful links.
+
+Tuning/Intervals
+================
+
+* Why There are Twelve Notes in Music: https://www.youtube.com/watch?v=IT9CPoe5LnM
+
+  * an `octave <https://en.wikipedia.org/wiki/Octave>`_
+    is an interval with a frequency ratio of 2:1 (e.g. A4 and A5)
+  * a `perfect fifth <https://en.wikipedia.org/wiki/Perfect_fifth>`_
+    is an interval with a frequency ratio of 3:2
+  * the `Pythagorean tuning <https://en.wikipedia.org/wiki/Pythagorean_tuning>`_
+    uses the 3:2 ratio
+  * the `Equal temperament <https://en.wikipedia.org/wiki/Equal_temperament>`_
+    divides the octave in 12 equal steps
+  * if we build a scale by using a 3:2 ratio, the frequencies for the octaves
+    don't match
```

Comment:
> the frequencies for the octaves don't match

Maybe clarify what "don't match" means.

Reply:
I tried to clarify it, but it's difficult to summarize it in one line.  The video and the wiki links cover it in more details.